server/rest/twofishes.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import ast
+import json
+import StringIO
+
+import geojson
+import requests
+from shapely.wkt import loads
+
+from girder.api import access
+from girder.api.describe import Description
+from girder.api.rest import Resource
+from girder.utility.model_importer import ModelImporter
+from girder.plugins.minerva.rest.geojson_dataset import GeojsonDataset
+from girder.plugins.minerva.utility.minerva_utility import findDatasetFolder
+
+
+class TwoFishes(Resource):
+    """Resource that handles geocoding related operations"""
+    def __init__(self):
+        self.resourceName = 'minerva_geocoder'
+        self.route('GET', ('autocomplete',), self.autocomplete)
+        self.route('GET', ('geojson',), self.getGeojson)
+        self.route('POST', ('geojson',), self.postGeojson)
+
+    @staticmethod
+    def getWktFromTwoFishes(twofishes, location):
+        """Gets wkt from twofishes for a given location"""
+        r = requests.get(twofishes,
+                         params={'query': location,
+                                 'responseIncludes': 'WKT_GEOMETRY'})
+        wkt = r.json()['interpretations'][0]['feature']['geometry']['wktGeometry']
+
+        return wkt
+
+    @staticmethod
+    def createGeometryFromWkt(wkt):
+        """Creates a shapely geometry from wkt"""
+        return loads(wkt)
+
+    @staticmethod
+    def createGeojson(twofishes, locations):
+        """Create geojson for given locations and twofishes url"""
+
+        try:
+            locations = ast.literal_eval(locations)
```

Comment:
When the user passes a list of inputs through swagger we get the string representation of the list. It is there to construct the list from the string.